### ISSUE TITLE
Uncaught Error: Cannot find module 'jmp'Require stack:- $ATOM_HOME/packages\Hydrogen\dist\zmq-k...

### ISSUE THREAD

**Issue by jacopoto** [nteract/hydrogen#2083]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.56.0 x64
**Electron**: 9.4.4
**OS**: Microsoft Windows 10 Home
**Thrown From**: [Hydrogen](https://github.com/nteract/hydrogen) package 2.15.0

### Stack Trace

Uncaught Error: Cannot find module 'jmp'
Require stack:
- C:\Users\Jacopo\.atom\packages\Hydrogen\dist\zmq-kernel.js
- C:\Users\Jacopo\.atom\packages\Hydrogen\dist\main.js
- C:\Users\Jacopo\AppData\Local\atom\app-1.56.0\resources\app.asar\static\index.html

```
At internal/modules/cjs/loader.js:800

Error: Cannot find module 'jmp'
Require stack:
- C:\Users\Jacopo\.atom\packages\Hydrogen\dist\zmq-kernel.js
- C:\Users\Jacopo\.atom\packages\Hydrogen\dist\main.js
- C:\Users\Jacopo\AppData\Local\atom\app-1.56.0\resources\app.asar\static\index.html
    at Module._resolveFilename (internal/modules/cjs/loader.js:797:17)
    at o._resolveFilename (electron/js2c/renderer_init.js:43:689)
    at Function.get_Module._resolveFilename (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:157820)
    at Module.require (/app.asar/static/index.js:61:43)
    at require (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:149153)
    at /packages/Hydrogen/dist/zmq-kernel.js:7:15)
    at /packages/Hydrogen/dist/zmq-kernel.js:325:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:149837)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:153385)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:149153)
    at /packages/Hydrogen/dist/main.js:34:38)
    at /packages/Hydrogen/dist/main.js:447:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:149837)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:153385)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (internal/modules/cjs/helpers.js:74:18)
    at customRequire (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:1:802769)
    at requireModule (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:14:1126115)
    at Package.requireMainModule (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:3496608)
    at Package.activateConfig (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:3485896)
    at PackageManager.packageHasSettings (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:11:904517)
    at PackageCard.hasSettings (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:14:2647820)
    at PackageCard.updateSettingsState (~/AppData/Local/atom/app-1.56.0/resources/app/static/<embedded>:14:2638088)
```

### Commands

```
     -2:53.6.0 core:backspace (input.hidden-input)
```

### Non-Core Packages

```
git-plus 8.7.1 
Hydrogen 2.15.0 
language-latex 1.2.0 
language-markdown 0.37.0 
language-stata 1.7.3 
latex 0.50.2 
latex-autocomplete 1.1.1 
latex-wordcount 0.6.0 
new-tab 0.5.3 
pdf-view 0.72.0 
teletype 0.13.4 
```

**Comment by aminya** [comment]
This is a downloading issue. Could you run the following:

```
apm uninstall hydrogen
apm install hydrogen
```

**Comment by mmarras** [comment]
This is on:

    Atom    : 1.56.0
    Electron: 9.4.4
    Chrome  : 83.0.4103.122
    Node    : 12.14.1
    OS: Windows10 Enterprise

I have the same issue. Although apm reports done on the reinstall, `shift+enter` still throws the error:

    Cannot find module 'jmp'
    Require stack:
    - C:\Users\######\.atom\packages\Hydrogen\dist\zmq-kernel.js
    - C:\Users\######\.atom\packages\Hydrogen\dist\main.js
    - C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app.asar\static\index.html
    Hide Stack Trace
    Error: Cannot find module 'jmp'
    Require stack:
    - C:\Users\######\.atom\packages\Hydrogen\dist\zmq-kernel.js
    - C:\Users\######\.atom\packages\Hydrogen\dist\main.js
    - C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app.asar\static\index.html
        at Module._resolveFilename (internal/modules/cjs/loader.js:797:17)
        at o._resolveFilename (electron/js2c/renderer_init.js:43:689)
        at Function.get_Module._resolveFilename (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:157820)
        at Module.require (file:///C:/Users/######/AppData/Local/atom/app-1.56.0/resources/app.asar/static/index.js:61:43)
        at require (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:149153)
        at Object.<anonymous> (C:\Users\######\.atom\packages\Hydrogen\dist\zmq-kernel.js:7:15)
        at Object.<anonymous> (C:\Users\######\.atom\packages\Hydrogen\dist\zmq-kernel.js:325:3)
        at Module.get_Module._compile (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:149837)
        at Object.value [as .js] (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:153385)
        at Module.load (internal/modules/cjs/loader.js:815:32)
        at Module._load (internal/modules/cjs/loader.js:727:14)
        at Function.Module._load (electron/js2c/asar.js:769:28)
        at Module.require (file:///C:/Users/######/AppData/Local/atom/app-1.56.0/resources/app.asar/static/index.js:72:46)
        at require (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:149153)
        at Object.<anonymous> (C:\Users\######\.atom\packages\Hydrogen\dist\main.js:34:38)
        at Object.<anonymous> (C:\Users\######\.atom\packages\Hydrogen\dist\main.js:447:3)
        at Module.get_Module._compile (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:149837)
        at Object.value [as .js] (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:153385)
        at Module.load (internal/modules/cjs/loader.js:815:32)
        at Module._load (internal/modules/cjs/loader.js:727:14)
        at Function.Module._load (electron/js2c/asar.js:769:28)
        at Module.require (file:///C:/Users/######/AppData/Local/atom/app-1.56.0/resources/app.asar/static/index.js:72:46)
        at require (internal/modules/cjs/helpers.js:74:18)
        at customRequire (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:1:802769)
        at requireModule (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:14:1126115)
        at Package.requireMainModule (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:3496608)
        at Package.activateNow (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:3484592)
        at C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:14:1879
        at Function.simpleDispatch (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:1182710)
        at Emitter.emit (C:\Users\######\AppData\Local\atom\app-1.56.0\resources\app\static\<embedded>:11:1184151)

**Comment by Cian-Byrne** [comment]
I have the same issue and have tried to reinstall.

**Comment by aminya** [comment]
OK. Now fixed! Sorry, I had to regenerate the built files! They were outdated. 

Fixed in 2.15.1
